apps/wa/aat/01/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
```

Comment:
Folder structure not correct, review similar to https://github.com/hmcts/cnp-flux-config/tree/master/apps/neuvector/aat 

you are adding duplication here - review above example with base